<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingDetailsDialog</class>
 <widget class="QDialog" name="BookingDetailsDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="Zug">
      <attribute name="title">
       <string>Zug</string>
      </attribute>
      <widget class="QGroupBox" name="ZugGroupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>291</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>Buchungsdetails</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLineEdit" name="TrainFromStation">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>24</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>24</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>von</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="TrainDepTime">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>50</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Abfahrt</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="TrainToStation">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>75</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>75</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nach</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="TrainArrTime">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>101</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>101</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ankunft</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="TrainTicketType">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>126</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>126</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tickettyp</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="TrainConnectingStations">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>152</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Über</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="Flug">
      <attribute name="title">
       <string>Flug</string>
      </attribute>
      <widget class="QGroupBox" name="FlugGroupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Buchungsdetails</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>von</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="FlightFromDest"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>nach</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="FlightToDest"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Airline</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="FlightAirline"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Tickettyp</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="FlightBookingClass"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Hotel">
      <attribute name="title">
       <string>Hotel</string>
      </attribute>
      <widget class="QGroupBox" name="HotelGroupBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>231</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Buchungsdetails</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Hotel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="HotelName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Stadt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="HotelTown"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Zimmer</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="HotelRoomType"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Auto">
      <attribute name="title">
       <string>Mietwagen</string>
      </attribute>
      <widget class="QGroupBox" name="AutoGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Buchungsdetails</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>Firma</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="CarCompany"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>Abholort</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="CarPickuplocation"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>Rückgabeort</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="CarReturnlocation"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>Wagentyp</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="CarVehicleClass"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Reise Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Preis:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="price"/>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="fromDate"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="bookingId">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="toDate"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Von</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bis</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookingDetailsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookingDetailsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
